templates/zerver/help/zulip-on-android.md

```diff
@@ -0,0 +1,186 @@
+# Using Zulip on Android
+
+Zulip has an Android application, compatible with Android versions 3.2 and
+higher.
+
+It's available on the Google Play Store, and can be downloaded from
+[here](https://play.google.com/store/apps/details?id=com.zulip.android), or by
+clicking the badge below.
+
+[![Play Store bagde](/static/images/help/google-play-badge.png)](https://play.google.com/store/apps/details?id=com.zulip.android)
+
+## Initial setup
+### Connecting to a server
+
+When you open the app for the first time, or after you log out, a screen asking
+for your server's address appears. You have to write the server's URL in it, and
+press *Enter*.
+
+!!! tip ""
+    There's no need of adding `https://`: after hitting *Enter*, it will appear
```

Comment:
I think just add this line should suffice, "There's no need to add `https://` in case your server is running secure HTTP (HTTPS) protocol".